packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1180,6 +1193,21 @@ export function purposeServiceBuilder(
 
       const purposeToClone = await retrievePurpose(purposeId, readModelService);
 
+      const activeConsumerDelegation = purposeToClone.data.delegationId
+        ? await retrieveDelegation(
```

Comment:
If we decide to remove the `delegationId` from the purpose model (see Slack thread: https://buildo.slack.com/archives/C07U5M9SJ1Y/p1736783396238289) then here we will need to use one of the other readmodel methods (probably `getActiveConsumerDelegationByPurpose`) as done in other functions here